### PR TITLE
Interface for Select component extracted

### PR DESCRIPTION
**Pull request by YevheniiaSilbernagl**
- [X] By placing an `X` in the preceding checkbox, I verify that I have signed the [Contributor License Agreement](https://github.com/SeleniumHQ/selenium/blob/master/CONTRIBUTING.md#step-6-sign-the-cla)

This interface will allow to create custom realisation of like-select components, which actually do not have select tag inside.

### CODE REVIEW

**Comment by jtymoschuk** [comment]
could somebody help me to understand what is the problem with tests?

**Comment by lukeis** [comment]
It's not the tests, you have compilation issues. First you need to add an entry in the BUCK and build.desc in the same folder. Next you should make the methods on the new interface public.

**Comment by jtymoschuk** [comment]
@lukeis Thank you for your support, I've fixed that issues, but is still fails. Sems like in a py module, but I didn't change py code, could you please take a look once more time?

File: java/client/src/org/openqa/selenium/support/ui/build.desc [hunk 1]
```diff
@@ -31,6 +31,7 @@ java_library(name = "components",
 
 java_library(name = "elements",
   srcs = [
+    'ISelect.java',
```
Comment (lukeis) [inline java/client/src/org/openqa/selenium/support/ui/build.desc]: double quotes required

File: java/client/src/org/openqa/selenium/support/ui/Select.java [hunk 2]
```diff
@@ -59,20 +59,23 @@ public Select(WebElement element) {
    * @return Whether this select element support selecting multiple options at the same time? This
    *         is done by checking the value of the "multiple" attribute.
    */
+  @Override
```
Comment (lukeis) [inline java/client/src/org/openqa/selenium/support/ui/Select.java]: can you revert all the Overrides added? it's not required for implementing interfaces, only overriding methods extended.

**Comment by jtymoschuk** [comment]
@lukeis overrides removed, tests passed

**Pull request merged**